content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Contêineres de inicialização de depuração
```

Comment:
This translation is inaccurate. The page talks about how to debug init containers; therefore, this should be "Depurando Contêineres de Inicialização".